analysis.py

```diff
@@ -1,12 +1,26 @@
+from pathlib import Path
+
 import numpy as np
 from matplotlib import pyplot as plt
 
 x = np.linspace(0, 10, 1000)
 y = np.cos(x)
 
-fig, ax = plt.subplots()
-ax.plot(x, y)
-ax.set_xlabel("x")
-ax.set_ylabel("y")
+plots_dir = Path().cwd() / "plots"
+plots_dir.mkdir(exist_ok=True)
+
+# Madicken's favorite :P
+with plt.rc_context(rc={"image.cmap": "jet"}):
```

Comment:
```suggestion
with plt.rc_context(rc={"image.cmap": "plasma"}):
```

Reply:
You know, I *do* love jet a ton, but plasma is so beautiful and I think would highlight this data better.